src/applications/facility-locator/utils/helpers.js

```diff
@@ -162,3 +162,17 @@ export const formatOperatingHours = operatingHours => {
   // Return the formatted operating hours.
   return formattedOperatingHours;
 };
+
+/**
+ *
+ * @param {String} website for example google.com, https://va.gov/
+ *
+ * Uses a regular expression to find va.gov in the website domain
+ *
+ * @returns {boolean}
+ *
+ */
+export const hasVADomain = website => {
+  const regExp = /((https?:\/\/www.?|(https?:\/\/?))va\.gov(\/*)?)/g;
```

Comment:
Why does this have the global (/g) flag?
Also why `(https?:\/\/www.?|(https?:\/\/?))`

Is this a better option?
```
https?:\/\/(www\.)?va\.gov(\/*)?
```

Reply:
this will only work on
```
https://www.va.gov
http://www.va.gov
https://va.gov
http://va.gov
```
but it will not work for any other subdomain on va.gov (staging.va.gov)

Reply:
```
https?:\/\/(:?.*\.)?va\.gov(\/*)?
```
will catch
```
https://www.va.gov
http://www.va.gov
https://va.gov
http://va.gov
https://staging.va.gov
http://staging.va.gov
https://qa.va.gov
http://qa.va.gov
```

Reply:
@darkbushido, this is exactly what we need - only work on 
https://www.va.gov
http://www.va.gov
https://va.gov
http://va.gov

Reply:
but you are right we need the to work on those subdomains too - https://staging.va.gov/pittsburgh-health-care/locations/pittsburgh-va-medical-center-university-drive/